### ISSUE TITLE
Formally incorporate tudatpy changelog into the news(?) workflow

### ISSUE THREAD

**Issue by jo11he** [tudat-team/tudatpy#45]
I send a changelog of the changes introduced by the `refactor/estimation_documentation` branch. 
We probably have a way to handle this properly now, probably through the news workflow(?)

@FilippoOggionni can you take care of that pls?

**Comment by jo11he** [comment]
changelog is in the ppt I send you on slack

**Comment by jo11he** [comment]
analogously for changes to tudat maybe, but I did not want to open a separate issue for that (also only two entries in the changelog)

**Comment by FilippoOggionni** [comment]
Yes, I will take care of it or at least keep it under my radar (both for tudat and tudatpy). The news workflow is currently on hold (see #46), but thanks for writing the changelog! 🔝
